### PR TITLE
Skip TensorFlow dependent tests if Tensorflow is not installed

### LINKED ISSUE #106: Skip Tensorflow dependent tests if Tensorflow is not installed
Skip Tensorflow dependent tests if Tensorflow is not installed.

### PR DESCRIPTION
**Pull request by dvreed77**
Skip TensorFlow dependent tests

fixes #106

### CODE REVIEW

**Comment by rwedge** [comment]
I'm doing some similar work on featuretools right now

EvalML did some work like this previously: https://github.com/alteryx/evalml/issues/2922

I think there should be a way to run the unit tests without tensorflow installed, but the tensorflow tests should run (and fail) if tensorflow is not installed

File: nlp_primitives/conftest.py [hunk 1]
```diff
@@ -1,13 +1,28 @@
...
 @pytest.fixture(autouse=True)
 def add_primitives(doctest_namespace, universal_sentence_encoder):
     doctest_namespace['universal_sentence_encoder'] = universal_sentence_encoder
+
+
+def pytest_ignore_collect(path):
+    skip_files = [
+        "universal_sentence_encoder.py",
+        "elmo.py"
+    ]
```
[the remark this answers is not in the document]
Comment (rwedge) [reply nlp_primitives/conftest.py:23]: then `pytest -m "not tensorflow"` will run the tests that don't have a tensorflow mark

[the remark this answers is not in the document]
Comment (dvreed77) [reply nlp_primitives/conftest.py:23]: I understand now

[the remark this answers is not in the document]
Comment (rwedge) [reply nlp_primitives/conftest.py:26]: went with "--notensorflow" flag

File: nlp_primitives/conftest.py [hunk 2]
```diff
@@ -1,13 +1,31 @@
...
 
-from nlp_primitives.universal_sentence_encoder import UniversalSentenceEncoder
+from nlp_primitives.tensorflow.universal_sentence_encoder import (
+    UniversalSentenceEncoder
+)
 
 
 @pytest.fixture(scope="session")
-def universal_sentence_encoder():
+def universal_sentence_encoder(pytestconfig):
```
Comment (rwedge) [inline nlp_primitives/conftest.py:9]: is passing in `pytestconfig` still necessary?

File: nlp_primitives/tensorflow/universal_sentence_encoder.py [hunk 3]
```diff
@@ -13,7 +13,7 @@ class UniversalSentenceEncoder(TransformPrimitive):
 
     Examples:
         >>> sentences = ["I like to eat pizza", "The roller coaster was built in 1885.", ""]
-        >>> # universal_sentence_encoder = UniversalSentenceEncoder()  # normal syntax
+        >>> #universal_sentence_encoder = UniversalSentenceEncoder()  # normal syntax
```
Comment (rwedge) [inline nlp_primitives/tensorflow/universal_sentence_encoder.py:16]: revert?

File: nlp_primitives/tests/test_elmo.py [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import numpy as np
 import pandas as pd
 
-from ..elmo import Elmo
+from ..tensorflow.elmo import Elmo
```
Comment (rwedge) [inline nlp_primitives/tests/test_elmo.py:4]: Can we update test_universal_sentence_encoder.py to use this style of import?

**Comment by dvreed77** [comment]
Fixes #107

**Pull request merged**